openpype/hosts/houdini/plugins/publish/validate_sop_output_node.py

```diff
@@ -81,3 +93,117 @@ def get_invalid(cls, instance):
                 "Output node `%s` has no geometry data." % output_node.path()
             )
             return [output_node.path()]
+
+    @classmethod
+    def repair(cls, instance):
```

Comment:
This seems quite magical. A lot of different cases that might occur. Are we sure this is something you might want to rely on as an artist. I think the only one I like is the repair where it uses your current selection. If the repair label would then be "Set selected node as output node" that'd be super clear to the end user. For example:

```
class SetSelectedAction(RepairAction):
    label = "Set selected node as output node"
```

And then using that as an action.